src/test/java/seedu/address/model/ModuleListTest.java

```diff
@@ -36,65 +37,65 @@ public void resetData_null_throwsNullPointerException() {
     }
 
     @Test
-    public void resetData_withValidReadOnlyAddressBook_replacesData() {
-        ModuleList newData = getTypicalAddressBook();
+    public void resetData_withValidReadOnlyModuleList_replacesData() {
+        ModuleList newData = getTypicalModuleList();
         moduleList.resetData(newData);
         assertEquals(newData, moduleList);
     }
 
     @Test
-    public void resetData_withDuplicatePersons_throwsDuplicatePersonException() {
-        // Two persons with the same identity fields
-        Person editedAlice = new PersonBuilder(ALICE).withTags(VALID_TAG_HUSBAND)
+    public void resetData_withDuplicateModules_throwsDuplicateModuleException() {
+        // Two modules with the same name
+        Module editedCS2030 = new ModuleBuilder(CS2030).withZoomLink(VALID_ZOOMLINK_CS2103T)
                 .build();
-        List<Person> newPersons = Arrays.asList(ALICE, editedAlice);
-        AddressBookStub newData = new AddressBookStub(newPersons);
+        List<Module> newModules = Arrays.asList(CS2030, editedCS2030);
+        ModuleListStub newData = new ModuleListStub(newModules);
 
-        // assertThrows(DuplicatePersonException.class, () -> moduleList.resetData(newData));
+        assertThrows(DuplicatePersonException.class, () -> moduleList.resetData(newData));
     }
 
     @Test
-    public void hasPerson_nullPerson_throwsNullPointerException() {
-        // assertThrows(NullPointerException.class, () -> moduleList.hasPerson(null));
+    public void hasModule_nullModule_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> moduleList.hasModule(null));
     }
 
     @Test
-    public void hasPerson_personNotInAddressBook_returnsFalse() {
-        // assertFalse(moduleList.hasPerson(ALICE));
+    public void hasModule_moduleNotInModuleList_returnsFalse() {
+        assertFalse(moduleList.hasModule(CS2030));
     }
 
     @Test
-    public void hasPerson_personInAddressBook_returnsTrue() {
-        // oduleList.addPerson(ALICE);
-        // assertTrue(moduleList.hasPerson(ALICE));
+    public void hasModule_moduleInModuleList_returnsTrue() {
+        moduleList.addModule(CS2030);
+        assertTrue(moduleList.hasModule(CS2030));
     }
 
     @Test
-    public void hasPerson_personWithSameIdentityFieldsInAddressBook_returnsTrue() {
-        // moduleList.addPerson(ALICE);
-        Person editedAlice = new PersonBuilder(ALICE).withTags(VALID_TAG_HUSBAND)
+    public void hasModule_moduleWithSameNameModuleList_returnsTrue() {
```

Comment:
This test is not updated for additional info added to Module like Grade Tracker which I recently added. I will update this in the future.

Reply:
Yes, sure. If the test doesn't work, we can always comment out first then make the changes later.